config/src/parameters/user/boilerplate.rs

```diff
@@ -184,7 +184,7 @@ impl FromEnv for RootPartial {
 #[serde(deny_unknown_fields, default)]
 pub struct GenesisPartial {
     pub public_key: UserField<PublicKey>,
-    pub private_key: UserField<PrivateKey>,
+    pub public_key_algorithm: UserField<Algorithm>,
```

Comment:
The public key is represented as a multihash, which includes the algorithm in it. So, the algorithm could be derived directly from `PublicKey`, therefore `public_key_algorithm` field is redundant.

Please remove.